draft-ietf-tls-subcerts.md

```diff
@@ -518,6 +520,21 @@ certificate satisfies the following criteria:
   {{RFC5280}}).
 
 
+# Operational Considerations
+
+## Client Clock Skew
+
+One of the risks of deploying a short-lived credential system based
+on absolute time is client clock skew.  If a client's clock is sufficiently
+ahead or behind of the server's clock, then credentials that are valid
+from the server's perspective will be rejected by the client.  For deployment
+scenarios where clock skew is an issue, it may be preferable to issue
```

Comment:
this seems like a very strange solution to me. Could we just say that this is a problem and leave it at that. 

Adding

"Clock skew also affects the validity of the original certificates. Thus this would affect delegated credentials, if the clock skew is smaller than the certificate lifetime. The lifetime of the DC should be set taking clock skew into account."

Reply:
This points out something I didn't address, which is the shortest viable lifetime of the DC. This is a valuable point to make, I'll add it.  However, the point I was trying to make here is that you need to manage rotating the DC in active use in the same manner that you would have to with certs with respect to clock skew. Specifically, you shouldn't use certs that are within some delta of the start or end of their validity period. Maybe this simplified text covers both cases, but I think it could be clearer.

Reply:
+1 to Subodh's suggestion -- let's just note the problem.

Reply:
How's the updated text, @siyengar @chris-wood ?